linode/manager
# fix: [M3-6998] - Disabled Select tooltip hover

## Description 📝
Fix the help icon tooltip being disabled when the Select component is disabled

TODO:
- [x] Remove test code before merging

## Preview 📷
| Before  | After   |
| ------- | ------- |
| <video src="https://github.com/linode/manager/assets/115299789/7dbc439f-43b5-43a4-b083-047d3b93a592" /> | <video src="https://github.com/linode/manager/assets/115299789/a08a0a19-f5d3-4ff5-a44e-bddcbb751192" /> |

## How to test 🧪
- Go to `/linodes/create` and hover over the tooltip icon for the disabled Region select; you should still be able to see the tooltip text
- Comment out the styling changes and notice the tooltip being unhoverable
